x-pack/test/alerting_api_integration/observability/custom_threshold_rule/avg_us_fired.ts

```diff
@@ -98,7 +97,7 @@ export default function ({ getService }: FtrProviderContext) {
           params: {
             criteria: [
               {
-                aggType: CUSTOM_AGGREGATOR,
+                aggType: 'custom',
```

Comment:
I kept this one in the test to check if the rule works as expected if `aggType: 'custom'` field exists.